gcp-instance-reaper/README.md

```diff
@@ -0,0 +1,38 @@
+This workflow looks at all of the GCP instances in a given account and zone and
+selects a subset of those to terminate. The termination criteria are:
+
+* Not tagged with a termination date or lifetime after 4 minutes
+* The `termination_date` or `lifetime` labels are present but cannot be parsed
+* The `termination_date` or `lifetime` labels indicate that the instance has
+  expired
+
+An instance may be configured to never terminate if its `lifetime` tag has the
+special value `indefinite`.
+
```

Comment:
The most recent feedback I got from Dex (Replicated) was it wasn't clear to him that the termination criteria is actually evaluating the *value* of the tag `lifetime` or `termination_date`, not just the *existence* of the tag. Can you specify the value format expected by the tag?